docs/src/main/sphinx/installation/cli.rst

```diff
@@ -262,3 +281,65 @@ To view debug information, including the stack trace for failures, use the
     ...
     at java.base/java.lang.Thread.run(Thread.java:834)
     select count(*) from tpch.tiny.nations
+
+============================================================== ======================================================================
```

Comment:
Oh .. get rid of this in this PR .. add it to a separate PR please and use list-table syntax

Reply:
And add it in a separate section called


Options


with some intro sentence like

The following table lists all supported command line options for the CLI in alphabetical order.